_zh-cn/overviews/reflection/typetags-manifests.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: multipage-overview
+title: TypeTags and Manifests
+partof: reflection
+overview-name: Reflection
+
+num: 5
+
+languages: [ja, zh-cn]
+permalink: /overviews/reflection/:title.html
+---
+
+与其他JVM语言一样，Scala的类型在运行时被擦除。这意味着，如果要检查某个实例的运行时类型，则可能无法访问Scala编译器在编译时可用的所有类型信息。
+
+如`scala.reflect.Manifest`，`TypeTags`可以看作是将编译时可用的所有类型信息携带到运行时的对象。
+例如，`TypeTag[T]`封装了某个编译时类型`T`的运行时类型表示。但是请注意，`TypeTag`应该被认为是对2.10之前的`Manifest`概念的更丰富的替代品，后者还与Scala反射完全集成。
+
+有三种不同类型的类型标记：
+
+1. `scala.reflect.api.TypeTags#TypeTag`。
+Scala类型的完整类型描述符。例如，`TypeTag[List[String]]`包含所有类型信息，在本例中是类型`scala.List[String]`。
+
+2. `scala.reflect.ClassTag`。
+Scala类型的部分类型描述符。例如，`ClassTag[List[String]]`只包含已删除的类类型信息，在本例中为`scala.collection.immutable.List`。`ClassTag`只提供对类型的运行时类的访问。类似于`scala.reflect.ClassManifest`。
+
+3. `scala.reflect.api.TypeTags#WeakTypeTag`.
+抽象类型的类型描述符（参见下面相应的小节）。
+
+## 获取`TypeTag`
+
+与`Manifest`类似，`TypeTag`总是由编译器生成，可以通过三种方式获得。
+
+### 通过方法`typeTag`、`classTag`或`weakTypeTag`
+
+通过使用通过`Universe`提供的方法`typeTag`，就可以直接获得特定类型的`TypeTag`。
+
+例如，要获取表示`Int`的`TypeTag`，我们可以执行以下操作：
+
+    import scala.reflect.runtime.universe._
+    val tt = typeTag[Int]
+
+或者类似地，要获得表示`String`的`ClassTag`，我们可以执行以下操作：
+
+    import scala.reflect._
+    val ct = classTag[String]
+
+这些方法中的每个方法都为给定的类型参数`T`构造一个`TypeTag[T]`或`ClassTag[T]`。
+
+### 使用类型为`TypeTag[T]`、`ClassTag[T]`或`WeakTypeTag[T]`的隐式参数
+    
+与`Manifest`一样，实际上可以 _请求_ 编译器生成`TypeTag`。这只需指定一个类型为`TypeTag[T]`的隐式 _票据_ 参数即可完成。如果编译器在隐式搜索期间找不到匹配的隐式值，它将自动生成一个`TypeTag[T]`。
```

Comment:
```suggestion
与`Manifest`一样，实际上可以 _请求_ 编译器生成`TypeTag`。这只需指定一个类型为`TypeTag[T]`的隐式 _例证_ 参数即可完成。如果编译器在隐式搜索期间找不到匹配的隐式值，它将自动生成一个`TypeTag[T]`。
```

/cc @scala/docs-zh `evidence`的翻译我不晓得有没有标准的译法。我建议翻译为“例证”。

Reply:
What is the original sentence? I'd say use `证据`

Reply:
> As with `Manifest`s, one can in effect _request_ that the compiler generate a
`TypeTag`. This is done by simply specifying an implicit _evidence_ parameter
of type `TypeTag[T]`. If the compiler fails to find a matching implicit value
during implicit search, it will automatically generate a `TypeTag[T]`.

In other words, it's using using the term evidence as a metaphor of typeclass instance. In Haskell for example this may come up as a phrase _equality evidence_. How would you say that in Chinese? I feel like 相等的证据 fits better than others?

Reply:
“例证” has the connotation that it's an _instance_ of possible proofs. But I agree `证据` might be easier to understand than inventing a new technical term.